### PR TITLE
cookiecutter: doc files

### PR DESCRIPTION
**Pull request by cadenmyers13**

### CODE REVIEW

**Comment by cadenmyers13** [comment]
@sbillinge cookiecutter pt2. This is just adding the doc directory

File: doc/Makefile [hunk 1]
```diff
@@ -0,0 +1,194 @@
+# Makefile for Sphinx documentation
```
Comment (sbillinge) [inline doc/Makefile:1]: we seem to have two Makefiles now.  We probably don't need this one (also hte make.bat in here) but look through it as if you were merging this Makefile and the one in source from cookiecutter, and raise any aquestions that come up.

**Review by sbillinge — COMMENTED** [review]
this looks good.  I am guessing that images would probably go into `_static`.  Also th esrc/diffpy/pdfgui/doc

File: doc/Makefile (hunk 1 above)
Comment (cadenmyers13) [reply doc/Makefile:1]: Generally I've been basing the structure off diffpy.pdfmorph and diffpy.utils. They both have `Makefile` directly under `doc/`. Also, diffpy.pdfmorph still has the make.bat file so wasn't sure if it was still needed here.

**Review by sbillinge — COMMENTED** [review]
something still seems to be wrong here.  we don't want doc/source hanging off the `_static`.  The `_staic` directory is a place where we put all the static files that are needed for the rest. It is kind of like a database where we put images and style files and things we need to build the html later.

A better workflow here would be to initiate a discussion with a small group of people to decide where to put everything.  It seems to me that we might want to just have tutorial hanging off doc and all the files needed for tutorial in the tutorial place for example.  Include Andrew in that convo as he has good input.

If we have the convo and decide everything, then you can just execute on a clear PR and we can merge it.

**Comment by sbillinge** [comment]
please see comments in the review.

**Comment by Sparks29032** [comment]
> this looks good. I am guessing that images would probably go into `_static`. Also th esrc/diffpy/pdfgui/doc

Shoot, missed this. Keep the images in manual, those are images for building the manual specifically. Some of these are generated by the manual scripts that screenshot latex equations for the html to display properly.

**Pull request closed without merging**